### PR TITLE
:bug: Fix csctl builder-image version

### PR DESCRIPTION
**Pull request by michal-gubricky**
**What this PR does / why we need it**:
Fix csctl builder-image version in `.builder-image-version.txt` file

### CODE REVIEW

**Review by kranurag7 — APPROVED** [review]
LGTM, Let me run renovate again after this gets merged. 

Hopefully renovate should bump now. 

```bash
$ crane ls ghcr.io/sovereigncloudstack/csctl-builder
0.2.0
```

**Comment by janiskemper** [comment]
@kranurag7 plz merge if this is ready

**Pull request merged**
